### PR TITLE
Rollup of several docs and build fixes.

### PR DESCRIPTION
**Pull request by theopolis**
1. Update contributing with examples on rebasing/squashing.
2. Add some missing CLI flags to the wiki.
3. Add the OS X version to the kernel extension package. To be safe from a feature-enablement perspective, encourage folks to use the package version corresponding to their OS. In practice the worst that has been observed is losing feature capability (e.g., no more MAC auth).
4. Include the systemd service for RHEL7 systems too.

### CODE REVIEW

**Comment by osqueryer** [comment]
Build (ffed82d1e57740ff6ea72a1816d421b03d8a162a) failed :-1:

**Comment by theopolis** [comment]
ok to test

**Comment by osqueryer** [comment]
Build (ffed82d1e57740ff6ea72a1816d421b03d8a162a) is good! :+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,13 +58,97 @@ Visit https://github.com/facebook/osquery and use the web UI to create a Pull Re
 
 ## Pull Request workflow
 
+In most cases your PR should represent a single body of work. It is find to change unrelated small-things like nits or code-format issues but make every effort to submit isolated changes. This makes documentation, references, regression tracking and if needed, a revert, easier.
```
Comment (theopolis) [inline CONTRIBUTING.md]: `/find/fine/`

**Comment by osqueryer** [comment]
Build (ffed82d1e57740ff6ea72a1816d421b03d8a162a) failed :-1:

**Pull request merged**